tiledb/sm/storage_manager/context.cc

```diff
@@ -124,6 +125,24 @@ stats::Stats* Context::stats() const {
   return stats_.get();
 }
 
+std::string Context::abs_path(const std::string& path) {
+  if (storage_manager_ != nullptr) {
+    bool found = false;
+    std::string local_root_path =
+        storage_manager_->config().get("vfs.local_root_path", &found);
+    if (found && local_root_path.length() > 0) {
+      return VFS::abs_path(path, local_root_path);
+    }
+  }
+  return path;
+}
```

Comment:
Appears that in the event there is no configured vfs.local_root_path, 
this is not going to return an absolute path (unless the incoming just happened to already be absolute.)
Not seeing any unit tests for Context::abs_path() additions here and below that might cover this situation...?

Reply:
unit test is in unit-cppapi-array.cc

Reply:
And, while I didn't look at the unit tests, I can assume they aren't actually testing for those possible error conditions.

There should be a test for a 'root_path' that is -not- actually a root path, 
and , unless you choose to rename the routine, a test to be sure an abs_path is returned when the provided 'path' is not absolute -and- no 'local_root_path' is configured.